build.sbt

```diff
@@ -223,24 +193,13 @@ lazy val algebird = Project(id = "algebird", base = file("."))
     mimaFailOnNoPrevious := false
   )
   .aggregate(
-    {
-      // workaround: https://github.com/sbt/sbt/issues/4181, simple workaround .settings(crossScalaVersions := List()) doesn't work
-      val projects =
```

Comment:
`algebirdSpark` was not aggregated properly, hence now the scalafmt and scalafix changes